api/api.go

```diff
@@ -83,17 +98,17 @@ func ListStateStats(w http.ResponseWriter, r *http.Request, d *db.Database) {
 // GetState provides information on a State
 func GetState(w http.ResponseWriter, r *http.Request, d *db.Database) {
 	w.Header().Set("Access-Control-Allow-Origin", "*")
-	st := util.TrimBasePath(r, "api/state/")
+	lineage := util.TrimBasePath(r, "api/state/")
```

Comment:
Shouldn't this be `/api/lineage` now?

Reply:
So I don't really know, for me this endpoint as well as `api/state/compare/` are always used to perform operations on states, lineages are only one parameter used.
But if you find it more suitable to replace state by lineage, no problem.

Reply:
I guess that's two different ways of organizing the API. Any preference @cryptobioz ?